articles/quickstart/native/android-vnext/00-login.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Login
+description: This tutorial demonstrates how to add user login to an Android application using Auth0.
+seo_alias: android
+budicon: 448
+topics:
+  - quickstarts
+  - native
+  - android
+github:
+  path: 00-Login
+contentType: tutorial
+useCase: quickstart
+---
+<!-- markdownlint-disable MD002 MD041 -->
+
+<%= include('../_includes/_getting_started', { library: 'Android' }) %>
+
+<%= include('../../../_includes/_callback_url') %>
+
+::: note
+If you are following along with the sample project you downloaded from the top of this page, you should set the **Allowed Callback URL** to  `demo://${account.namespace}/android/YOUR_APP_PACKAGE_NAME/callback`.
+:::
+
+Replace `YOUR_APP_PACKAGE_NAME` with your application's package name, available as the `applicationId` attribute in the `app/build.gradle` file.
+
+<%= include('../../../_includes/_logout_url', { returnTo: 'demo://' + account.namespace + '/android/YOUR_APP_PACKAGE_NAME/callback' }) %>
+
+Replace `YOUR_APP_PACKAGE_NAME` with your application's package name, available as the `applicationId` attribute in the `app/build.gradle` file.
+
+## Install the Auth0 Android SDK
+
+Add the [Auth0 Android](https://github.com/auth0/Auth0.Android) SDK into your project. The library will make requests to the Auth0's Authentication and Management APIs.
+
+### Add Auth0 to Gradle
+
+In your app's `build.gradle` dependencies section, add the following:
+
+```groovy
+apply plugin: 'com.android.application'
+android {
+  // ...
+}
+dependencies {
+  // Add the Auth0 Android SDK
+  implementation 'com.auth0.android:auth0:1.+'
+}
+```
+
+::: note
+If Android Studio lints the `+` sign, or if you want to use a fixed version, check for the latest in [Maven](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22auth0%22%20g%3A%22com.auth0.android%22) or [JCenter](https://bintray.com/auth0/android/auth0).
+:::
+
+::: panel Sync Project with Gradle Files
+Remember to synchronize using the Android Studio prompt or run `./gradlew clean build` from the command line. For more information about Gradle usage, check [their official documentation](http://tools.android.com/tech-docs/new-build-system/user-guide).
+:::
+
+Add manifest placeholders required by the SDK. The placeholders are used internally to define an `intent-filter` that captures the authentication callback URL.
+
+To add the manifest placeholders, add the next line:
+
+```xml
+// app/build.gradle
+
+apply plugin: 'com.android.application'
+compileSdkVersion 30
+android {
+    defaultConfig {
+        applicationId "com.auth0.samples"
+        minSdkVersion 21
+        targetSdkVersion 30
+        // ...
+
+        // ---> Add the next line
+        manifestPlaceholders = [auth0Domain: "@string/com_auth0_domain", auth0Scheme: "demo"]
+        // <---
+    }
+}
```

Comment:
@lbalmaceda Can you check the versions here are correct?

Reply:
They are correct 👍